### ISSUE TITLE
`no-array-sort` should not trigger on oneliner arrow functions

### ISSUE THREAD

**Issue by mkahvi** [sindresorhus/eslint-plugin-unicorn#2734]
`no-array-sort` demands I make simple oneliner arrow function into messier full function or unnecessarily clone the array.

```js
const sortTuple = (arr) => arr.sort(([, a], [, b]) => collator.compare(a, b));
```
